packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/ManageTopicsSection/ManageTopicsSection.tsx

```diff
@@ -86,6 +88,9 @@ export const ManageTopicsSection = () => {
           buttonWidth={isWindowExtraSmall ? 'full' : 'narrow'}
           buttonHeight="med"
           onClick={handleReversion}
+          disabled={filteredTopics.every(
```

Comment:
i have a feeling this will always be disabled?

Reply:
`topics` is reset when items in the draggable list are moved. However, `filteredTopics` matches what's on the database. Until the 'Save Changes' button is selected, these two things will remain different, i.e. the button will remain disabled *until* you drag something in the list.